loss-detection-app/res/device-mqtt-go/profiles/cv-roi-device.yaml

```diff
@@ -0,0 +1,19 @@
+# Copyright © 2022 Intel Corporation. All rights reserved.
+# SPDX-License-Identifier: BSD-3-Clause
+
+name: "device-cv-roi-mqtt"
+manufacturer: "Intel"
+model: "CV ROI events"
+labels:
+  - "cv"
+  - "roi"
+  - "enter"
+  - "exit"
+description: "A device for CV region of interest (ROI) events"
+
+deviceResources:
+  - name: "cv-roi-event"
+    description: "A CV ROI event"
+    properties:
+      valueType: "string"
```

Comment:
Should be using "Object" type now since the ROI is an JSON object. Should be in follow-up PR (separate for better naming PR) as the code will be impacted. This applies to all these device profiles.

https://intel-iot-devkit.github.io/rtsf-at-checkout-reference-design/rtsf_at_checkout_events/checkout_events/